src/test/rpc/Simulate_test.cpp

```diff
@@ -630,7 +630,8 @@ class Simulate_test : public beast::unit_test::suite
             auto validateOutput = [&](Json::Value const& resp,
                                       Json::Value const& tx) {
                 auto result = resp[jss::result];
-                checkBasicReturnValidity(result, tx, 5, "20");
+                checkBasicReturnValidity(
+                    result, tx, 5, env.current()->fees().base * 2);
```

Comment:
Please change 5 -> seq()

Reply:
fixed in 0bbf589f53a13069448d709faa5cea88b6ee547f